CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ for further information about branching and tagging conventions.
 
 #### Changed
 - Show only one QR code on Coinpayments deposit
+- Use global api.js in all modules
```

Comment:
an old api.js still used in deposit-fiat-card and verification/general-form

Reply:
ok